charts/datafold/charts/server/values.yaml

```diff
@@ -41,10 +41,10 @@ ingress:
 
 resources:
   limits:
-    memory: 3500Mi
+    memory: 5500Mi
   requests:
     cpu: 500m
-    memory: 3500Mi
+    memory: 5500Mi
```

Comment:
I calibrated memory to make pods fit in 64G nodes. That means we need to leave a bit of headroom. With 2x5500 we go over that, so we always run with 2 nodes.

Reply:
updated to 7000Mi and made replica count 1, keeping webWorkers at 4